staging/src/k8s.io/pod-security-admission/manifests/deployment.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: podsecurity-webhook-deployment
+  namespace: podsecurity-webhook
+  labels:
+    app: podsecurity-webhook
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: podsecurity-webhook
+  template:
+    metadata:
+      labels:
+        app: podsecurity-webhook
+    spec:
+      containers:
+        - name: podsecurity-webhook
+          image: k8s.gcr.io/sig-auth/podsecurity-webhook:latest
+          ports:
+            - containerPort: 80
```

Comment:
should this be a high port so the pod doesn't need net_bind_admin?

also, on a related note, should we make our pod spec compatible with the restricted profile ;-)